.github/CODEOWNERS

```diff
@@ -1,12 +1,15 @@
-/.github/       @centreon/centreon-ci
-/.git*          @centreon/centreon-ci
-/.project       @centreon/centreon-ci
-/Jenkinsfile    @centreon/centreon-ci
-/selinux/       @centreon/centreon-ci
-/project/       @centreon/centreon-ci
-*.sh            @centreon/centreon-ci
+/.github/       @centreon/centreon-devops
+/.git*          @centreon/centreon-devops
+/.project       @centreon/centreon-devops
+/Jenkinsfile    @centreon/centreon-devops
+/selinux/       @centreon/centreon-devops
+/project/       @centreon/centreon-devops
+*.sh            @centreon/centreon-devops
 
-*.po            @ghost
+/.snyk				@centreon/centreon-security
+/sonar-project.properties	@centreon/centreon-security
```

Comment:
It seems the team @centreon/centreon-security does ot exist

Reply:
@bdauria team must be created ? or those lines must be deleted ?

Reply:
the codeowner will be modified next week to reflect latest changes.
The security team will be created in october

Reply:
This will be handled later on